src/Pyz/Glue/Log/LogDependencyProvider.php

```diff
@@ -28,15 +28,23 @@ class LogDependencyProvider extends SprykerLogDependencyProvider
     protected function addLogHandlers(Container $container): Container
     {
         $container->set(static::LOG_HANDLERS, function () {
-            return [
-                new StreamHandlerPlugin(),
-                new ExceptionStreamHandlerPlugin(),
-            ];
+            return $this->getLogHandlers();
         });
 
         return $container;
     }
 
+    /**
+     * @return array<\Monolog\Handler\HandlerInterface>
+     */
+    protected function getLogHandlers(): array
```

Comment:
must be implemented in core first.https://github.com/spryker/spryker/blob/master/Bundles/Log/src/Spryker/Glue/Log/LogDependencyProvider.php